### DISCUSSION TITLE
Typing text in Rich Text Editor

### DISCUSSION THREAD

**Discussion by louisfosse** [AutomaApp/automa#194] [Q&A]
Hello,

I try to auto complete a form which contains a [TinyMce](https://www.tiny.cloud/tinymce/features/) rich text editor. 
All the selectors I tried don't work. I tried to switch to an iframe or tried to inject some text with Javascript code without any success. 
Has anyone faced this issue already? 

Thanks a lot!

**Comment by AI-Based-Businesses** [comment]
I have the same issue right now. Did you solve it??

**Comment by Delusionnnn** [comment]
Do the CSS selectors or XPATH selectors never find the element? I usually use the aria-label when I run into issues.
